ddtrace/tracer/tracer.go

```diff
@@ -347,6 +348,17 @@ func newTracer(opts ...StartOption) *tracer {
 		t.abandonedSpansDebugger = newAbandonedSpansDebugger()
 		t.abandonedSpansDebugger.Start(t.config.spanTimeout)
 	}
+	for name, conf := range c.integrations {
+		if !conf.Instrumented {
+			continue
+		}
+		v := conf.Version
+		if v == "" {
+			v = "unknown"
+		}
+		t.statsd.Incr("datadog.tracer.instrumentations", []string{fmt.Sprintf("instrumentation:%s", name), fmt.Sprintf("instrumentation_version:%s", v)}, 1)
```

Comment:
[I think] fmt.Sprintf is a more complex operation than simple string concatenation, meaning there is a slight performance hit. So, when applicable (e.g. a simple case like this), might as well use concatenation:
```suggestion
		t.statsd.Incr("datadog.tracer.instrumentations", []string{"instrumentation:" + name, "instrumentation_version:" + v}, 1)
```